src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Calculator implements ArithmeticOperations {
+    @Override
+    public double calculator(int firstElement, int secondElement, char operator) {
+        switch (operator) {
+            case ('+') :
+                long result = (long) firstElement + (long) secondElement;
```

Comment:
```suggestion
                long result = (long) firstElement +  secondElement;
```
Тебе достаточно скастить 1 элемент.